src/windows.zig

```diff
@@ -0,0 +1,35 @@
+const std = @import("std");
+const w = std.os.windows;
+
+const c = @cImport({
+    @cInclude("windows.h");
+});
+
+pub const NULL = c.NULL;
+pub const FILE_MAP_READ = c.FILE_MAP_READ;
+
+pub extern "kernel32" fn CreateFileA(
+    lpFileName: w.LPCSTR, // I tried to avoid this with usingnamespace std.os.windows but it didn't work so idk
```

Comment:
This comment, I tried `usingnamespace std.os.windows;` but I couldn't seem to use these types in the func signatures so I just gave up and did this